File: QuanLyHienMauApi/QuanLyHienMauApi/Models/CaNhan.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace QuanLyHienMauApi.Models
{
    public class CaNhan
    {
        [Key]
        [ForeignKey("ThongTinTaiKhoan")]
        public int ID { get; set; }

        public string? NhomMau { get; set; }
        public string? HeRh { get; set; }

        public DateTime? NgaySinh { get; set; }
        public int SoLanHien { get; set; } = 0;
        public DateTime? NgayHienGanNhat { get; set; }
        public string? AnhXacMinh { get; set; }
        public int TrangThaiXacMinh { get; set; } = 0;
        public int? ChieuCao { get; set; }     
        public double? CanNang { get; set; }
        public virtual NguoiDung? ThongTinTaiKhoan { get; set; }
    }
}

File: HienMau/app/src/main/java/com/example/hienmau/models/CaNhan.java
package com.example.hienmau.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class CaNhan implements Serializable {
    @SerializedName("ID")
    public int id;

    @SerializedName("NhomMau")
    public String nhomMau;

    @SerializedName("HeRh")
    public String heRh;

    @SerializedName("NgaySinh")
    public String ngaySinh;

    @SerializedName("SoLanHien")
    public int soLanHien;

    @SerializedName("ChieuCao")
    public Integer chieuCao;

    @SerializedName("CanNang")
    public Double canNang;
    @SerializedName("NgayHienGanNhat")
    public String ngayHienGanNhat;
}
